src/v2/api/index.md

```diff
@@ -1582,41 +1582,41 @@ type: api
 
 ### vm.$once( event, callback )
 
-- **Arguments:**
+- **Argumentos:**
   - `{string} event`
   - `{Function} callback`
 
 - **Uso:**
 
-  Listen for a custom event, but only once. The listener will be removed once it triggers for the first time.
+  Escucho un evento personalizado, pero solamente una vez. La escucha será removida una vez que es disparada por primera vez.
 
 ### vm.$off( [event, callback] )
 
-- **Arguments:**
-  - `{string | Array<string>} event` (array only supported in 2.2.2+)
+- **Argumentos:**
+  - `{string | Array<string>} event` (arreglo solamente es soportado en 2.2.2+)
   - `{Function} [callback]`
 
 - **Uso:**
 
-  Remove custom event listener(s).
+  Remover escuchas de evento(s) personalizado(s).
```

Comment:
I agree with you it's difficult to translate this in a way it sounds natural. Maybe what we can do is translate it as "escuchas" but add `(_listener_)` next to it? 🤔

Reply:
I like that idea! @FLarra 😄